controller/share_manager_controller.go

```diff
@@ -225,6 +269,7 @@ func (c *ShareManagerController) Run(workers int, stopCh <-chan struct{}) {
 	for i := 0; i < workers; i++ {
 		go wait.Until(c.worker, time.Second, stopCh)
 	}
+	go c.runLeaseCheck(stopCh)
```

Comment:
Do we have any mechanism to ensure the recovery of a goroutine if it somehow crashes?

Reply:
Can you give more details on how disk monitor recover? From a quick reading, I don't quite to get it yet

Reply:
Oh, I was wrong... bad memory.